database/migrations/2018_04_12_185714_add_connections_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddConnectionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('connections', function (Blueprint $table) {
+            $table->unsignedInteger('owner_id');
```

Comment:
$table->increments('id').

We want a primary key here.

Reply:
as discussed